packages/@o3r/telemetry/README.md

```diff
@@ -18,3 +18,6 @@ A set of helpers to retrieve tool usage metrics.
 
 https://github.com/AmadeusITGroup/otter/blob/main/docs/telemetry/PRIVACY_NOTICE.md
 
+## Additional information
+
+By default, the project name send will be the name of the package. It can be override by providing `config.o3rProjectName` in the `package.json`.
```

Comment:
```suggestion
By default, the project name sent will be the name of the package. It can be overridden by providing `config.o3rProjectName` in the `package.json`.
```